app/views/organisations/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :title, "Departments, agencies and public bodies - GOV.UK" %>
+
+<h1>Departments, agencies and public bodies - GOV.UK</h1>
+<br>
+<h2>Ministerial departments</h2>
+<%= @content.select { |item| item[:department_type] == "Ministerial" }.count %>
+<ul>
+  <% @content.each do |item| %>
+    <% if item[:department_type] == "Ministerial" %>
+    <li class="app-c-taxon-list__item"><%= link_to item[:title], item[:base_path] %></li>
+    <% end %>
+    <% if item[:agencies].length > 1 && item[:department_type] == "Ministerial" %>
```

Comment:
If would suggest moving some of this logic out of the view and into a presenter to make things a bit tidier